test/tests/constraints/equal_value_embeded_constraint/1D_2D_kinematic.i

```diff
@@ -0,0 +1,92 @@
+###########################################################
+# This is a test that demonstrates a user-defined
+# constraint. It forces variables in overlapping portion of
+# two blocks to have the same value
+#
+# @Requirement F1.90
```

Comment:
Drop these requirements lines. We aren't tracking requirements this way any more. We are putting this information in the test specification now:
See the auto-generated table here:
http://mooseframework.org/moose/sqa/moose_rtm.html

You can hover over individual requirements to see how you are supposed to annotate them.